engines/python/setup/djl_python/neuron_utils/model_loader.py

```diff
@@ -301,7 +301,6 @@ def partition(self, save_path: str, **kwargs):
         self.model_config.save_pretrained(save_path)
         self.model = self.load_hf_model()
         self.load_path = self.get_load_path()
-        save_pretrained_split(self.model, self.load_path)
```

Comment:
I think we still want to save this in the partition method, lets move it from the self.load_path as the save_pretrained_split, to instead self.split_model_path. I do think this also means that we will need to change our copy behavior on the docker partition side. We want the checkpoint directory to have the split model weights directory, the original model config.json, and the generation_config.json if it exists.

Reply:
To further qualify this, we want it to be run if we have not already run this in the load_inf2_model_from_disk method.